src/main/scala/rules/SymbolicExecutionRules.scala

```diff
@@ -29,14 +29,20 @@ trait SymbolicExecutionRules extends Immutable {
       val model = v.decider.getModel()
       if (model != null && !model.contains("model is not available")){
         val nativeModel = Model(model)
-        var ce: Counterexample = if (Verifier.config.counterexample.toOption.get == "native") {
-          val oldHeap = if (s.oldHeaps.contains(Verifier.PRE_STATE_LABEL))
-            Some(s.oldHeaps(Verifier.PRE_STATE_LABEL).values)
-          else None
-          SiliconNativeCounterexample(s.g, s.h.values, oldHeap, nativeModel)
-        }else{
-          SiliconVariableCounterexample(s.g, nativeModel)
+        val ce_type = Verifier.config.counterexample.toOption.get

```

Comment:
I believe `config.counterexample()` is recommended over `config.counterexample.toOption.get`